### PR TITLE
Create order

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

File: server/reducer.js [hunk 1]
```diff
@@ -6,10 +6,40 @@ function setState(state, newState) {
   return Object.assign({}, state, newState);
 }
 
+function toggleOrder(state, id) {
```
Comment (achiu8) [inline server/reducer.js]: Why are we adding this reducer?

File: server/seed.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  }).save((err, item) => {
+    Order({
+      id: '18',
+      status: constants.CLOSED,
+      items: item._id
+    }).save();
+  });
+
+  Item({
+    name: 'Bun Bo Hue',
```
Comment (achiu8) [inline server/seed.js:27]: Noice.

File: src/action_creators.js [hunk 3]
```diff
@@ -22,8 +22,17 @@ export function loadOrders() {
   };
 }
 
+export function addOrder(order) {
```
Comment (achiu8) [inline src/action_creators.js:25]: Does this action actually write to the db yet?

File: src/components/NewOrder.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { Component, DOM as dom } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../action_creators';
+
+const mapStateToProps = function (state) { return {
```
Comment (achiu8) [inline src/components/NewOrder.js]: Can we put the return on next line?

File: src/components/NewOrder.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+};
+
+class NewOrder extends Component {
+  render() {
+    return dom.div(
+      { className: 'container' },
+      dom.h1(null, 'New Order Page'),
+      dom.button({
+          className: 'btn btn-default',
+          onClick: () => {
```
Comment (achiu8) [inline src/components/NewOrder.js]: I think this can just be `onClick: this.props.addOrder`.

File: server/reducer.js (hunk 1 above)
Comment (kprakobkit) [inline server/reducer.js]: my bad. i think it was a bad merge...you removed it right?

File: src/action_creators.js (hunk 3 above)
Comment (kprakobkit) [inline src/action_creators.js:25]: not yet.

**Pull request merged**